modules/meeting/app/services/meetings/ical_helpers.rb

```diff
@@ -89,8 +89,8 @@ def ical_datetime(time, timezone_id)
       Icalendar::Values::DateTime.new time.in_time_zone(timezone_id), "tzid" => timezone_id
     end
 
-    def ical_organizer(meeting)
-      Icalendar::Values::CalAddress.new("mailto:#{meeting.author.mail}", cn: meeting.author.name)
+    def ical_organizer
+      Icalendar::Values::CalAddress.new("mailto:#{Setting.mail_from}", cn: Setting.app_title)
```

Comment:
What happens if `Setting.mail_from` is empty or invalid email address?
I tried setting empty value and invalid value and it happily saves it. Currently it's not enforced to be a valid value.

![image](https://github.com/user-attachments/assets/bf749d10-e770-45e8-8efb-baf1671269ef)